BNN.py

```diff
@@ -67,12 +78,13 @@ def new_bnn(log_divergence_weight=-10, prior_stddev=1, residual_weight=0.1):
   predictions_list_trainset.append(y)
 
 predictions_list_trainset = np.asarray(predictions_list_trainset)
-print(predictions_list_trainset.shape)
-print(predictions_list_trainset.mean())
-print((predictions_list_trainset.std(0, ddof=1)).mean())
+
+print("Predictions shape", predictions_list_trainset.shape)
+print("Predictions mean", predictions_list_trainset.mean())
+print("Predictions shape", predictions_list_trainset.std(0, ddof=1))
```

Comment:
I think you meant "Predictions std"

```suggestion
print("Predictions shape", predictions_list_trainset.std(0, ddof=1))
```